docs/development.md

```diff
@@ -75,7 +75,7 @@ first-time setup.
 4.  Install the Python dependencies:
 
     ```bash
-    $ uv sync --dev
+    $ uv sync
```

Comment:
I think we should keep this explicit with `--dev`, to ensure those are installed

Reply:
Oh yea, missed that